app/models/candidate_interface/work_experience_form.rb

```diff
@@ -32,9 +34,9 @@ def self.build_from_experience(work_experience)
         start_date_day: work_experience.start_date.day,
         start_date_month: work_experience.start_date.month,
         start_date_year: work_experience.start_date.year,
-        end_date_day: work_experience.end_date.day,
-        end_date_month: work_experience.end_date.month,
-        end_date_year: work_experience.end_date.year,
+        end_date_day: work_experience.end_date.nil? ? '' : work_experience.end_date.day,
```

Comment:
The following **should** do the same thing and be a bit more concise?
```suggestion
        end_date_day: work_experience.end_date&.day || ''
```

Reply:
Updated in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/438/commits/464651dbb7618b111c5dd43e7d56dc845e82f077!